test/agoneum/web/controllers/registration_controller_test.exs

```diff
@@ -1,30 +1,40 @@
 defmodule Agoneum.Web.RegistrationControllerTest do
   use Agoneum.Web.ConnCase
 
-  alias Agoneum.Account
+  describe "UI registration" do
+    @create_attrs %{email: "some email", name: "some name", password: "some password"}
+    @invalid_attrs %{email: nil, name: nil, password: nil}
 
-  @create_attrs %{email: "some email", name: "some name", password: "some password"}
-  @invalid_attrs %{email: nil, name: nil, password: nil}
+    test "creates user and redirects to the main page", %{conn: conn} do
+      conn = post conn, registration_path(conn, :create), user: @create_attrs
+      assert redirected_to(conn) == page_path(conn, :index)
+    end
 
-  def fixture(:user) do
-    {:ok, user} = Account.create_user(@create_attrs)
-    user
+    test "does not create user and renders errors when data is invalid", %{conn: conn} do
+      conn = post conn, registration_path(conn, :create), user: @invalid_attrs
+      assert html_response(conn, 200) =~ "Sign Up"
```

Comment:
The test passes, so it seems correct. What I can do is `put_status` on the error case so we get a non-200 code